PCAxis.Web.Controls/SearchValues/SearchValuesCodebehind.vb

```diff
@@ -390,12 +390,12 @@ Public Class SearchValuesCodebehind
         SearchResults.Items.Clear()
 
         For Each v As Value In v_final
-            Select Case Me.Marker.Variable.PresentationText
-                Case 0
-                    liS = New ListItem(v.Code, v.Code)
-                Case Else
-                    liS = New ListItem(v.Text, v.Code)
-            End Select
+            If Marker.AlwaysShowCodeAndTextInSearchResult Then
+                liS = New ListItem(v.CodeAndValue, v.Code)
+            Else
+                liS = New ListItem(v.Text, v.Code)
+            End If
+
             liS.Selected = True
```

Comment:
new ListItem(v.Code, v.Code) is not very usefull, but when ignoring Me.Marker.Variable.PresentationText we change how the system behaves even if the new setting is false. 
Is this on purpose?

Reply:
Excelent!